### PR TITLE
ROC-5421 Defendant phone number by defendant

### PR DESCRIPTION
**Pull request by karthikkannankainos**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-5421

### Change description
Defendant phone number by defendant.

### CODE REVIEW

File: src/main/features/response/routes/your-details.ts [hunk 1]
```diff
@@ -65,7 +66,7 @@ export default express.Router()
     ErrorHandling.apply(async (req: express.Request, res: express.Response): Promise<void> => {
       const form: Form<PartyDetails> = req.body
 
-      if (form.hasErrors()) {
+      if (form. hasErrors()) {
```
Comment (robertparkinson) [inline src/main/features/response/routes/your-details.ts]: extra white space here that needs to be removed

File: src/main/features/response/views/check-and-send.njk [hunk 2]
```diff
@@ -26,7 +26,22 @@
         }
       )
     }}
-    {{ row(label = 'Contact number (optional)', value = draft.defendantDetails.mobilePhone.number, changeLink = ResponsePaths.defendantMobilePage.evaluateUri({ externalId: claim.externalId }), bold = true) }}
+
+    {% if claim.claimData.defendant.phone %}
+      {{ row(
```
Comment (robertparkinson) [inline src/main/features/response/views/check-and-send.njk]: As this is on the same page as the confirm your name and address details section it should be included in there. So there should be no change link and it should only have a bottom border and remove the border from the other section above.

File: src/main/features/response/views/your-details.njk [hunk 3]
```diff
@@ -58,7 +64,9 @@
               heading: addressSectionHeading,
               name: 'address'
             },
-            showCorrespondenceAddressSection = true
+            showCorrespondenceAddressSection = true,
+            postcodeLabel = 'dd',
```
Comment (robertparkinson) [inline src/main/features/response/views/your-details.njk]: why is this being set to 'dd'?

Comment (karthikkannankainos) [reply src/main/features/response/views/your-details.njk]: removed.

**Comment by robertparkinson** [comment]
@sudheer-ajax there is a PR to refactor out mobile from the code base here: https://github.com/hmcts/cmc-citizen-frontend/pull/1745

**Pull request merged**